index.bs

```diff
@@ -525,7 +527,10 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 6. Initialize |issuedRequests| to an empty list.
 
-7. For each authenticator currently available on this platform, perform the following steps:
+7. If no authenticator is available, reject |promise| with a <a>DOMException</a> whose name is "NotFoundError", and 
+    terminate this algorithm.
+
+8. For each authenticator currently available on this platform, perform the following steps:
```

Comment:
This is https://github.com/tabatkins/bikeshed/issues/796, which we haven't yet gotten around to implementing. :-/